### ISSUE TITLE
2.2.0-RC2 missing latest commits from master

### ISSUE THREAD

**Issue by MustafaHaddara** [fakemongo/fongo#343]
Pull request https://github.com/fakemongo/fongo/pull/334 was merged on Apr 23. 

Release version [2.2.0-RC2](https://github.com/fakemongo/fongo/releases/tag/fongo-2.2.0-RC2) was release May 9.

However, 2.2.0-RC2 is based on https://github.com/fakemongo/fongo/commit/090330b65c9c33d333b39b49af280abf7fe1867b, from the [driver36 branch](https://github.com/fakemongo/fongo/tree/driver36), whereas PR #334 went into master and those commits never got merged into the driver36 branch. 

Can you push a release to maven-central with the changes from #334? I've been waiting on that fix to go public.

**Comment by twillouer** [comment]
can you check against the driver37 branch ?

**Comment by MustafaHaddara** [comment]
How do I convince maven to look at the driver37 branch?

**Comment by twillouer** [comment]
git clone ...

mvn clean install -DskipTests

<dependency>
	<groupId>com.github.fakemongo</groupId>
	<artifactId>fongo</artifactId>
	<version>2.2.0-RC3-SNAPSHOT</version>
	<scope>test</scope>
</dependency>

**Comment by MustafaHaddara** [comment]
It still fails with `2.2.0-RC3-SNAPSHOT`. I still get an `InsertManyWriteConcernException` (ie. what I fixed in https://github.com/fakemongo/fongo/pull/334)

```
com.mongodb.InsertManyWriteConcernException
	at com.mongodb.FongoBulkWriteCombiner.throwOnError(FongoBulkWriteCombiner.java:98)
	at com.mongodb.FongoDBCollection.executeBulkWriteOperation(FongoDBCollection.java:1218)
	at com.mongodb.DBCollection.executeBulkWriteOperation(DBCollection.java:2352)
	at com.mongodb.BulkWriteOperation.execute(BulkWriteOperation.java:121)
```

Also, my project is currently using `mongo-java-driver 3.6.3` and `morphia 0.108`. I had to bump my `mongo-java-driver` version to `3.7.0` just to get the test case to start, but that will not be sustainable for the rest of my project; I need to fix in the version for the 3.6 driver.

**Comment by MustafaHaddara** [comment]
@twillouer any updates on this?

**Comment by jmmut** [comment]
@MustafaHaddara I think the branch driver36 was rebased in June 6th to include your changes and remain on mongo 3.6 (branch driver36 is currently on 75f7204), but it wasn't retagged nor deployed in maven central. 

I was also trying to use #334, ideally if current master (fa0d307) was deployed as 2.1.2 or similar. I tried 2.1.2-SNAPSHOT compiled locally and it worked as expected.

@twillouer , any chance these deployments can be done?
